portal/views/partials/address/address-tokens.njk

```diff
@@ -71,3 +69,33 @@
     </div>
   </div>
 </div>
+
+
+<div class="block-section address-tokens is-mobile">
+  <div class="block-section__header--cols">
+	<h2><strong>{{ _('filter.token') }}:</strong> ({{ address.balances.length }} {{ _('helper.field.total') }})</h2>
+  </div>
+  <div class="block-section__content filter-form">
+  <span>Select Token</span>
+	<select name="txTokens" id="txTokens" class="selectpicker" data-width="100%" data-size="5">
+      <option value="" selected>{{ _('filter.default.selectAll') }}</option>
+        {% for balanceInfo in address.balances %}
+          {% if balanceInfo.assetId === opts.token %}
```

Comment:
the Option block is repeated , find a way to add the selected property dynamically: maybe like:
<option value="" {{ balanceInfo.assetId === opts.token ? 'selected' :''}}">>
There should also only be one selected option